frontend/projects/abtesting/src/app/features/dashboard/home/components/experiment-post-condition/experiment-post-condition.component.ts

```diff
@@ -14,6 +17,7 @@ export class ExperimentPostConditionComponent implements OnInit, OnChanges {
   @Input() experimentInfo: ExperimentVM;
   @Input() newExperimentData: Partial<ExperimentVM>;
   @Output() emitExperimentDialogEvent = new EventEmitter<NewExperimentDialogData>();
+  @Input() experiment: ExperimentVM;
```

Comment:
What's the purpose of taking experiment having `ExperimentVM` as you already have experimentInfo with the same type which you are getting from the parent component?

Reply:
removed